scripts/updateOldInTechOpenLinks.py

```diff
@@ -42,8 +41,10 @@ def main():
 
             if match == None:
                 doabProcess.importSingleOAIRecord(record.source_id)
-                
-            
+                doabProcess.saveRecords()
+                #Make session record a set after updating record in database to reduce memory space
+                record = set()
```

Comment:
should be `doabProcess.records = set()` here! That will make sure that the process class doesn't build up a huge object containing all the records we've processed.

Reply:
Appreciate the correction!